internal/monitors/prometheusexporter/prometheus.go

```diff
@@ -56,6 +56,9 @@ type Config struct {
 	// Path to the client TLS key to use for TLS required connections
 	ClientKeyPath string `yaml:"clientKeyPath"`
 
+	// Number of seconds before timing out while requesting data from target host.
```

Comment:
A better doc for the config option is something like this: https://github.com/signalfx/signalfx-agent/blob/master/internal/monitors/forwarder/monitor.go#L30